Makefile

```diff
@@ -257,31 +340,14 @@ endif
 # https://github.com/operator-framework/community-operators/blob/7f1438c/docs/packaging-operator.md#updating-your-existing-operator
 .PHONY: catalog-build
 catalog-build: opm ## Build a catalog image.
-	$(OPM) index add --container-tool $(IMAGE_BUILDER)  --mode semver --tag $(CATALOG_IMG) --bundles $(BUNDLE_IMGS) $(FROM_INDEX_OPT)
+	$(OPM) index add --container-tool docker --mode semver --tag $(CATALOG_IMG) --bundles $(BUNDLE_IMGS) $(FROM_INDEX_OPT)
```

Comment:
Not sure about this one, I wonder whether it should be `$(CONTAINER_TOOL)`. However, operator-sdk used "docker" here, but I guess this is a mistake.

@VaishnaviHire do we even need the `catalog-build` target as we probably never will have a own catalog entry (but be part of ODH) ?

Reply:
I've opened https://github.com/operator-framework/operator-sdk/pull/6932 to address this upstream.

Reply:
Thanks I think its okay to remove olm targets. But it can be a follow-up to this PR.